lineapy/cli/cli.py

```diff
@@ -59,6 +59,13 @@
     help="Print out logging for graph creation and execution.",
     is_flag=True,
 )
+@click.version_option(
+    None,
+    "--version",
+    "-v",
```

Comment:
The documentation I was reading says

https://click.palletsprojects.com/en/8.1.x/api/

> param_decls ([str](https://docs.python.org/3/library/stdtypes.html#str)) – One or more option names. Defaults to the single value "--version".

I opted to specify both since we want both.